starter/angular-xeditable/js/xeditable.js

```diff
@@ -1040,6 +1052,11 @@ angular.module('xeditable').factory('editableFormController',
     }
   });
  
+  $rootScope.$on('closeEdit', function() {
```

Comment:
allows closing all 'edits' from outside of the directive.. .useful when you are using the xEditable system with other controls that 'open and close"